docs/sources/datasources/aws-cloudwatch/aws-authentication.md

```diff
@@ -0,0 +1,111 @@
++++
+title = "Authentication"
+description = "AWS authentication"
+keywords = ["grafana", "aws", "authentication"]
+aliases = ["/docs/grafana/latest/datasources/cloudwatch"]
+weight = 205
++++
+
+# AWS authentication
+
+There are three different authentication methods available. 
+
+`AWS SDK Default` performs no custom configuration at all and instead uses the [default provider](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html) as specified by the AWS SDK for Go. This requires you to configure your AWS credentials separately, such as if you've [configured the CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html), if you're [running on an EC2 instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html), [in an ECS task](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html) or for a [Service Account in a Kubernetes cluster](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
+
+`Credentials file` corresponds directly to the [SharedCredentialsProvider](https://docs.aws.amazon.com/sdk-for-go/api/aws/credentials/#SharedCredentialsProvider) provider in the Go SDK. In short, it will read the AWS shared credentials file and find the given profile. While `AWS SDK Default` will also find the shared credentials file, this option allows you to specify which profile to use without using environment variables. It doesn't have any implicit fallbacks to other credential providers, and will fail if using credentials from the credentials file doesn't work.
```

Comment:
```suggestion
`Credentials file` corresponds directly to the [SharedCredentialsProvider](https://docs.aws.amazon.com/sdk-for-go/api/aws/credentials/#SharedCredentialsProvider) provider in the Go SDK. In short, it will read the AWS shared credentials file and find the given profile. While `AWS SDK Default` will also find the shared credentials file, the `Credentials file` option allows you to specify which profile to use without using environment variables. This option doesn't have any implicit fallbacks to other credential providers, and will fail if the credentials provided from the file aren't correct.
```

Reply:
For line 13, I would use a bulleted list (and assume the reader can count). It is what I learned from tech writers way back in the day, but perhaps guidelines have changed since then.

Reply:
I would go further and remove "three" in the preceding sentence because it is a bug waiting to happen when we add or remove an option and fail to update the number of ways. Rather, how about "There is more than one way to ..." and list them in the order that we recommend to introduce a tendency for the reader to chose the top choice.

Reply:
Good points, thanks for the review.